rele/worker.py

```diff
@@ -17,7 +18,18 @@ class Worker:
     :param subscriptions: list :class:`~rele.subscription.Subscription`
     """
 
-    def __init__(self, subscriptions, gc_project_id, credentials, default_ack_deadline):
+    def __init__(self, subscriptions, *args):
+        if args[0] == rele.config:
```

Comment:
Im not sure how this is passing since `rele.config` is a module and `args[0]` is an instance of a `Config` class. I wonder if it has to do with what [the test](https://github.com/mercadona/rele/pull/119/files#diff-3ba621217225120eff2c061666b8043cR68) is asserting. 

Perhaps we should do:

```suggestion
        if isinstance(args[0], Config):
```

And in addition add some explicit assertions in the test, like:

```
assert worker.credentials == ...

```